api-report/merge-tree.api.md

```diff
@@ -305,6 +320,18 @@ export interface IAttributionCollection<T> {
     splitAt(pos: number): IAttributionCollection<T>;
 }
 
+// @internal @sealed (undocumented)
+export interface IAttributionCollectionSerializer {
+    populateAttributionCollections(segments: Iterable<ISegment>, summary: SerializedAttributionCollection): void;
+    // Warning: (ae-forgotten-export) The symbol "SerializedAttributionCollection" needs to be exported by the entry point index.d.ts
```

Comment:
this bit isn't great. i'd love to not expose the serialized type, but if we want the user to import and specify the method of doing so (even if we implement that factory as is done in this PR), dropping the type here does lose some typesafety. If we did expose it it'd obviously be marked internal.